How-To-Contributing.md

```diff
@@ -0,0 +1,29 @@
+# Contributing
+
+If you wish to contribute to Cacti, the process is very easy.  The first step
+is to register for a GitHub account.  Once you have a GitHub account, you then
+need to learn about their community guidelines and familiarize yourself with
+how to create a pull request.  Pull requests are how we accept submissions from
+the community.
+
+There are a few key points.  First, if you want a feature, or find an issue,
+and the forums are leading your to GitHub, create an Issue, this issue can be
+tagged either an enhancement or a bug.  Regardless of what it is, if you think
+you can create a fix for that, prepare a pull request with your changes.
+
+We generally want you creating pull requests against the **develop** branch,
+but there may be times, where we either require a separate branch to be created
+for you, or as you to create a pull request in a separate branch.  Bottom line,
+ask questions, and follow the directions that the team provides you.
+
+[Standards-Patch-Creation](Standards-Patch-Creation.md)
```

Comment:
Maybe add some text rather than just a semi-random link.  E.g., 

```markdown
For more information on the Pull Request process see: [Standards-Patch-Creation](Standards-Patch-Creation.md)
```

Or better wording if you can find it.

Reply:
I'm pretty sure I did, maybe didn't commit it.